pkg/controller/deployment/deployment_controller.go

```diff
@@ -29,7 +29,7 @@ import (
 	"k8s.io/klog/v2"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Don't think this is needed

Reply:
goimport automatically added this :( reverted